content/200-orm/050-overview/500-databases/200-database-drivers.mdx

```diff
@@ -157,3 +157,24 @@ import { PrismaClient } from 'db'
 
 const client = new PrismaClient()
 ```
+
+### Driver adapters and specific frameworks
+
+#### Nuxt
+
+Using a driver adapter with [Nuxt](https://nuxt.com/) to deploy to an edge function environment does not work out of the box, but adding the `nitro.experimental.wasm` configuration option fixes that:
```

Comment:
```suggestion
Using a driver adapter with [Nuxt](https://nuxt.com/) and [Nitro](https://nuxt.com/docs/api/kit/nitro) to deploy to an edge function environment does not work out of the box, but adding the `nitro.experimental.wasm` configuration option fixes that:
```

Reply:
I think I would leave this out for simplicity. Users don't make a conscious decision to use Nitro, and will note via the `nitro.experimental...` if they are using the correct version which our instructions apply to. Agree?

Reply:
Ok, agree